texttospeech/package.json

```diff
@@ -11,14 +11,15 @@
     "node": ">=16.0.0"
```

Comment:
question: Is this still the min version?

Reply:
revved this, thanks for catching.